styles.css

```diff
@@ -1 +1,64 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+.container {
+    background-color: hsl(320, 10%, 70%);
+    display: grid;
+    row-gap: 16px;
+    column-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    grid-template-areas:
+    "title"
+    "main"
+    "c-picture"
+    "d-img-c-disc"
+    "e-text"
+    "dyn"
+    "footer";
```

Comment:
/* namen kan je gewoon letter van maken is korter in schrijven maar dit is ook oke*/